cpp/daal/src/algorithms/k_nearest_neighbors/kdtree_knn_classification_train_dense_default_impl.i

```diff
@@ -103,12 +130,28 @@ public:
     size_t size() const { return _count; }
 
 private:
+    void grow()
+    {
+        size_t newCapacity = (_capacity == 0 ? defaultSize : _capacity * 2);
+        T * newData        = static_cast<T *>(service_malloc<T, cpu>(newCapacity));
+
+        for (size_t i = 0; i < _count; ++i)
```

Comment:
Please use `daal_memcpy_s` if the typical size of the loop is big enough. I.e. 20+ iterations.